### PR TITLE
[improve][docbot] Move the preceding steps into the action

### LINKED ISSUE #67: Simplify invocation of docbot action
### Motivation
The [Current invocation of docbot](https://github.com/apache/pulsar/blob/46b1239a23fd2a250b009df1e9b4a65341d299a7/.github/workflows/ci-documentbot.yml):

```
jobs:
  label:
    steps:
      - name: Checkout action
        uses: actions/checkout@v3
        with:
          repository: apache/pulsar-test-infra
          ref: master

      - name: Set up Go
        uses: actions/setup-go@v3
        with:
          go-version: 1.18

      - name: Labeling
        uses: ./docbot
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          LABEL_WATCH_LIST: 'doc,doc-required,doc-not-needed,doc-complete'
          LABEL_MISSING: 'doc-label-missing'
```

These steps before Labeling should be organized into the action itself, the desired invocation should be illustrated below:

```
jobs:
  label:
    steps:
      - name: Labeling
        uses: pulsar/pulsar-test-infra/docbot@master
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          LABEL_WATCH_LIST: 'doc,doc-required,doc-not-needed,doc-complete'
          LABEL_MISSING: 'doc-label-missing'
```

inspired from https://github.com/apache/pulsar-test-infra/issues/58#issuecomment-1225889058

### PR DESCRIPTION
**Pull request by maxsxu**
Fixes #67

### CODE REVIEW

File: docbot/action.yml [hunk 1]
```diff
@@ -8,6 +8,16 @@ branding:
 runs:
   using: composite
   steps:
-    - run: go run main.go
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        repository: pulsar/pulsar-test-infra
```
Comment (tisonkun) [inline docbot/action.yml:14]:
```suggestion
        repository: apache/pulsar-test-infra
```
?

File: docbot/action.yml [hunk 2]
```diff
@@ -8,6 +8,16 @@ branding:
...
+        ref: master
```
Comment (tisonkun) [inline docbot/action.yml:15]:
```suggestion
```

File: docbot/action.yml (hunk 1 above)
Comment (maxsxu) [reply docbot/action.yml]: Fixed

File: docbot/action.yml (hunk 2 above)
Comment (maxsxu) [reply docbot/action.yml:15]: The current branch ref is master: https://github.com/apache/pulsar-test-infra/tree/master

Comment (tisonkun) [reply docbot/action.yml:15]: Fair enough. If we remove this config, the default branch is also master.

Comment (maxsxu) [reply docbot/action.yml]: Removed, let ref uses the default branch.

**Review by nodece — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]

**Review by urfreespace — APPROVED** [review]

**Review by Anonymitaet — APPROVED** [review]

**Comment by tisonkun** [comment]
**NOTE** With composite action, we always check out the master branch of `apache/pulsar-test-infra` to get the code. If we use Docker action, we can deliver the code along with each version of docbot.

Since we always use the master branch and actions in pulsar-test-infra are used for pulsar only by design, it's not a big deal. Just share the note here for your information.

**Pull request merged**